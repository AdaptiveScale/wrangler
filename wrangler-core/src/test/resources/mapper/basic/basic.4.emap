// This is a test for using functions and also assignments from already assigned values.
out.rec3 = in.rec3; // time();
out.a = in.rec3[0].a;
out.f = 1.23;
out.x = 3.4d;