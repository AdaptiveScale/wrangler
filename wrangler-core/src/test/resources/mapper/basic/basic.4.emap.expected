{
  "a": "rec3-array-a-1",
  "b": null,
  "c": null,
  "d": null,
  "e": null,
  "f": 1.23,
  "x": 3.4,
  "rec2a": null,
  "rec3": [
    {
      "a": "rec3-array-a-1",
      "b": 5
    },
    {
      "a": "rec3-array-a-2",
      "b": 6
    },
    {
      "a": "rec3-array-a-3",
      "b": 7
    },
    {
      "a": "rec3-array-a-4",
      "b": 8
    },
    {
      "a": "rec3-array-a-5",
      "b": 9
    }
  ]
}